sr_description/hand/xacro/finger/biotac/biotac.urdf.xacro

```diff
@@ -154,20 +154,20 @@ xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom">
         <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="0.0" iyz="0.0"
         izz="0.0" />
       </inertial>
-      <visual>
+      <!--visual>
```

Comment:
In an example of a hand with just the first finger with biotac, this was the tf tree before:
![framesoriginal](https://cloud.githubusercontent.com/assets/12493239/10311397/7290a70a-6c3e-11e5-931a-b9e9a3d912d2.png)

And this is how it looks now:
![frames](https://cloud.githubusercontent.com/assets/12493239/10311411/7a161a6e-6c3e-11e5-847e-2a2f5ff3689f.png)